Source/Eto.WinForms/Forms/Controls/GridHandler.cs

```diff
@@ -218,6 +218,12 @@ public override void AttachEvent(string id)
 				case Grid.CellClickEvent:
 					Control.CellClick += (sender, e) =>
 					{
+						if (e.RowIndex > -1)
```

Comment:
Will this will make the grid behave differently when the CellClick event is handled vs. not?